index.html

```diff
@@ -4,7 +4,18 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>IP Finder</title>
```

Comment:
I add this name intentionally. Don't return it to `Vite + React` please.

Reply:
done